### PR TITLE
Login Screen + Stack Screen

### PR DESCRIPTION
**Pull request by duongcscx**
- Login screen
- Login API
- Stack screen

### CODE REVIEW

**Review by lethuccrma — APPROVED** [review]

File: src/navigators/AppNavigator.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
   return (
     // <NavigationContainer>
-    <Stack.Navigator>
-      <Stack.Screen
-        name="LAUNCH_SCREEN"
-        component={LaunchScreen}
-        options={{ headerShown: false }}
-      />
-    </Stack.Navigator>
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
```
Comment (lytienphuc0810) [inline src/navigators/AppNavigator.js]: add a rule to eslint config

File: src/screens/LoginScreen.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    textAlign: 'center',
+    fontWeight: '500',
+    color: 'white',
+  },
+});
+
+function LoginScreen() {
+  const { setAuthentication } = useContext(AuthContext);
+  const navigation = useNavigation();
+  const [inputs, setInputs] = useState({
```
Comment (lytienphuc0810) [inline src/screens/LoginScreen.js]: use const [user, setUser]

Comment (duongcscx) [reply src/screens/LoginScreen.js]: Do you mean we should change the name of the variable (like [inputs, setInputs] to [user, setUser])?

Comment (lytienphuc0810) [reply src/screens/LoginScreen.js]: yes, should group the fields in to domain objects.

**Comment by lethuccrma** [comment]
Hi a @lytienphuc0810, all comments have been fixed.

**Review by lytienphuc0810 — APPROVED** [review]

**Pull request merged**